docs/articles/infrastructure/ci-cd/freeipa.rst

```diff
@@ -316,4 +484,87 @@ Try to identify an LDAP user:
+``ipa-server-install --uninstall`` a few times to clean the system. 2-3 should
+do it. This is to ensure all configurations are cleared if you tried installing
+multiple times. After that you need to re-enroll the client before continuing
+with ``ipa-client-install --enable-dns-updates --force-join`` (Don't forget to
+re-enable mkhomedir).
+
+If you find yourself re-installing FreeIPA server you will run into certificate
+problems when accessing the Web UI. This is because the certificate server
+doesn't randomize the ID:s of the generated certificates. So the browser won't
+accept seeing a different certificate with the same Domain, Subject and Serial
+Number. To mitigate this you can either pass the an argument during installation
+to change the Subject or configure the Dogtag server to randomize ID:s.
+
+Set the subject:
+
+.. code-block:: bash
+
+    ipa-server-install --setup-dns --hostname=ipa.example.com --realm=IPA.EXAMPLE.COM --domain=ipa.example.com --subject="O=EXAMPLE.COM 20181112"
+
+Or modify Dogtag through the following parameter in
+/etc/dogtag/tomcat/pki-tomcat/ca/default.cfg :
+
+.. code-block:: bash
+    :caption: Snippet from /etc/dogtag/tomcat/pki-tomcat/ca/default.cfg
+
+    [CA]
+    pki_random_serial_numbers_enable=True
+
+Also described on the Dogtag wiki [#dogtag_serial_numbers]_.
+
+If ipa-ca-fails configuring the CA instance with the following output:
+
+.. code-block:: none
+
+    ipa-ca-install
+    Directory Manager (existing master) password: 
+
+    Run connection check to master
+    Connection check OK
+    /usr/lib/python2.7/dist-packages/urllib3/connection.py:266: SubjectAltNameWarning: Certificate for replica1.example.com has no `subjectAltName`, falling back to check for a `commonName` for now. This feature is being removed by major browsers and deprecated by RFC 2818. (See https://github.com/shazow/urllib3/issues/497 for details.)
+      SubjectAltNameWarning
+    Configuring certificate server (pki-tomcatd). Estimated time: 3 minutes 30 seconds
+      [1/23]: creating certificate server user
+      [2/23]: creating certificate server db
+      [3/23]: setting up initial replication
+    Starting replication, please wait until this has completed.
+    Update in progress, 3 seconds elapsed
+    Update succeeded
+
+      [4/23]: creating installation admin user
+      [5/23]: setting up certificate server
+    ipa.ipaserver.install.cainstance.CAInstance: CRITICAL Failed to configure CA instance: Command '/usr/sbin/pkispawn -s CA -f /tmp/tmpkb9ORA' returned non-zero exit status 1
+    ipa.ipaserver.install.cainstance.CAInstance: CRITICAL See the installation logs and the following files/directories for more information:
+    ipa.ipaserver.install.cainstance.CAInstance: CRITICAL   /var/log/pki/pki-tomcat
+      [error] RuntimeError: CA configuration failed.
+
+It may mean that it cannot find java. You need to make sure /bin/java exists. If it doesn't, symlink it to the proper place. In Ubuntu 16.04:
```

Comment:
```suggestion
It may mean that it cannot find java. You need to make sure /bin/java exists. If it does not, symlink it to the proper place. In Ubuntu 16.04:
```